src/components/Box/BoxConstants.js

```diff
@@ -0,0 +1,262 @@
+import styles from "./Box.module.scss";
+
+export const DISABLED = Object.freeze({
+  DISABLED: styles.opacityDisabled
+});
+
+export const BORDER = Object.freeze({
```

Comment:
for all borders - either boolean or add layout-border-color

Reply:
is this note still relevant? @orrgottlieb @3dyonic ?